src/components/Event/EventContainer.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from "react";
+import EventBox, { type EventBoxProps } from "./EventBox";
+import { Faculties } from "./TestData";
+
+const DEFAULT_CHOSEN_FACULTY_VALUE = "คณะ / Faculty";
+
+interface EventContainerProps {
+  Events: EventBoxProps[];
+}
+
+const EventContainer = ({ Events }: EventContainerProps) => {
```

Comment:
same to https://github.com/isd-sgcu/oph66-frontend/pull/23/files#diff-38ae16de040485cdeeb1cd128e0e38036c457f4c89d37770ec12694ba9b82553R40